app/views/layouts/application.html.haml

```diff
@@ -5,6 +5,11 @@
     %meta{:name => 'viewport', :content => 'width=device-width, initial-scale=1'}/
     %meta{'http-equiv' => 'X-UA-Compatible', :content => 'IE=edge'}/
 
+    %link{:rel => "apple-touch-icon", :sizes => "180x180", :href => "/assets/apple-touch-icon.png"}/
+    %link{:rel => "manifest", :href => "/assets/manifest.json"}/
```

Comment:
Right... Actually, manifest.json links to some assets, but you have to be aware that they're compiled in the asset pipeline and the end result will have slightly different file names (with digests) which you can't know in advance. So manifest.json should probably be in the assets folder and be something like `manifest.json.erb` and link to the graphics using the `asset_path` helper.